src/org.xtuml.bp.mc/src/org/xtuml/bp/mc/AbstractProperties.java

```diff
@@ -1,10 +1,6 @@
-// ========================================================================
-//
-//File: $RCSfile: AbstractProperties.java,v $
-//Version: $Revision: 1.4 $
-//Modified: $Date: 2013/01/10 23:43:52 $
+//========================================================================
 //
-//(c) Copyright 2005-2014 by Mentor Graphics Corp. All rights reserved.
+//  File: AbstractProperties.java
 //
 //========================================================================
 //This document contains information proprietary and confidential to

```

Comment:
Nuke this line, too.  And I think we can stop providing a comment that simply names the file.